extensions/wikia/AdEngine/js/provider/directGptMobile.js

```diff
@@ -10,7 +10,7 @@ define('ext.wikia.adEngine.provider.directGptMobile', [
 		'mobile',
 		{
 			INVISIBLE_HIGH_IMPACT:      {size: '1x1'},
-			MOBILE_TOP_LEADERBOARD:     {size: '320x50,320x100,300x250,300x50,1x1'},
+			MOBILE_TOP_LEADERBOARD:     {size: '320x50,320x100,300x250,300x50,320x480,1x1'},
```

Comment:
Weren't we supposed to remove `1x1` or we did it in the Big Fancy Ad Below the fold story? ;)